notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    aiplatform.init(project=PROJECT_ID, staging_bucket=BUCKET_NAME)
</pre><p>Remove BUCKET_NAME parameter and add location parameter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>